ui-tests/src/test/java/io/jenkins/plugins/analysis/warnings/FreeStyleConfigurationUITest.java

```diff
@@ -0,0 +1,92 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import org.junit.Test;
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.po.FreeStyleJob;
+import io.jenkins.plugins.analysis.warnings.IssuesRecorder.TrendChartType;
+import static io.jenkins.plugins.analysis.warnings.Assertions.*;
+
+/**
+ * Acceptance tests for the Warnings Next Generation Plugin.
+
+ */
+@WithPlugins("warnings-ng")
+@SuppressWarnings({"checkstyle:ClassFanOutComplexity", "PMD.SystemPrintln", "PMD.ExcessiveImports"})
+public class FreeStyleConfigurationUITest extends AbstractJUnitTest {
+
+    private static final String PATTERN = "**/*.txt";
+    private static final String ENCODING = "UTF-8";
+    private static final String REFERENCE = "reference";
+    private static final String SOURCE_DIRECTORY = "relative";
+    private static final String SERVERITY = "NORMAL";
+
+    /**
+     * Verifies that job configuration screen correctly modifies the properties of an {@link IssuesRecorder} instance.
+     */
+    @Test
+    public void shouldSetPropertiesInJobConfiguration() {
+        FreeStyleJob job = jenkins.getJobs().create(FreeStyleJob.class);
+
+
+        IssuesRecorder issuesRecorder = job.addPublisher(IssuesRecorder.class, recorder -> {
+            recorder.setTool("Eclipse ECJ");
+        });
+
+        issuesRecorder.setSourceCodeEncoding(ENCODING);
+        issuesRecorder.setSourceDirectory(SOURCE_DIRECTORY);
+        issuesRecorder.setAggregatingResults(true);
+        issuesRecorder.setTrendChartType(TrendChartType.TOOLS_ONLY);
+        issuesRecorder.setBlameDisabled(true);
+        issuesRecorder.setForensicsDisabled(true);
+        issuesRecorder.setEnabledForFailure(true);
+        issuesRecorder.setIgnoreQualityGate(true);
+        issuesRecorder.setIgnoreFailedBuilds(true);
+        issuesRecorder.setFailOnError(true);
+        issuesRecorder.setReferenceJobField(REFERENCE);
+        issuesRecorder.setHealthReport(1, 9, SERVERITY);
+        issuesRecorder.setReportFilePattern(PATTERN);
+
+
+        job.save();
+        job.configure();
+        issuesRecorder.openAdvancedOptions();
+        assertThat(issuesRecorder.getSourceCodeEncoding()).isEqualTo(ENCODING);
```

Comment:
Custom assertions machen den Test besser lesbar:
```suggestion
       assertThat(issuesRecorder).hasSourceCodeEncoding(ENCODING);
```